charts/osm/crds/osmconfig.yaml

```
@@ -0,0 +1,117 @@
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: osmconfigs.config.openservicemesh.io
+spec:
+  group: config.openservicemesh.io
+  scope: Cluster
+  names:
+    kind: OsmConfig
+    listKind: OsmConfigList
+    shortNames:
+      - osmconfig
+    plural: osmconfigs
+    singular: osmconfig
+  versions:
+    - name: v1alpha1
+      served: true
+      storage: true
+      schema:
+        openAPIV3Schema:
+          type: object
+          properties:
+            spec:
+              type: object
+              properties:
+                sidecar:
+                  description: Configuration for Envoy sidecar
+                  type: object
+                  properties:
+                    enable_privileged_init_container:
+                      description: Enables privileged init containers for pods in mesh. When false, init containers only have NET_ADMIN.
+                      type: boolean
+                      default: false
+                    log_level:
+                      description: Sets the logging verbosity of Envoy proxy sidecar, only applicable to newly created pods joining the mesh.
+                      type: string
+                      default: "error"
+                      enum:
+                        - trace
+                        - debug
+                        - info
+                        - warning
+                        - warn
+                        - error
+                        - critical
+                        - off
+                traffic:
+                  description: Configuration for traffic management
+                  type: object
+                  properties:
+                    enable_egress:
+                      description: Enables egress in the mesh
+                      type: boolean
+                      default: false
+                    outbound_ip_range_exclusion_list:
+                      description: Global list of IP address ranges to exclude from outbound traffic interception by the sidecar proxy.
+                      type: string
+                      pattern: ((?:\d{1,3}.){3}\d{1,3})\/(\d{1,2})(,((?:\d{1,3}.){3}\d{1,3})\/(\d{1,2}))*$
+                    use_https_ingress:
+                      description: Enable HTTPS ingress on the mesh
+                      type: boolean
+                      default: false
+                    enable_permissive_traffic_policy_mode:
+                      description: True for allowing all traffic into the mesh. False  enabling allow-all mode in the mesh i.e. no traffic policy enforcement in the mesh. If set to false, enables deny-all traffic policy in mesh i.e. an SMI Traffic Target is necessary for services to communicate.
```

The description is incorrect regarding traffic being allowed into the mesh.

```suggestion
                      description: True for allowing traffic to flow between client and service pods within the mesh without SMI traffic policies, i.e. no traffic policy enforcement in the mesh. If set to false, enables deny-all traffic policy in mesh i.e. an SMI Traffic Target is necessary for services to communicate.
```